data/shubhampatil9500.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Shubham Patil",
+  "bio": "Devops Engineer **AWS Cloud**.  ",
+  "links": [
+    {
+      "name": "Follow me on LinkedIN",
```

Minor typo on line 6. Consider updating LinkedIN to LinkedIn.
```suggestion
      "name": "Follow me on LinkedIn",
```